### ISSUE TITLE
Invalid enum values for list workflow runs

### ISSUE THREAD

**Issue by miyajan** [octokit/routes#664]
- endpoint scope: Actions
- endpoint name: List workflow runs
- route: /repos/:owner/:repo/actions/workflows/:workflow_id/runs
- documentation URL: https://developer.github.com/v3/actions/workflow_runs/#list-workflow-runs

### Current specification

https://github.com/octokit/routes/blob/250db518150bc12a19155ed7dcfcccb82d35396e/openapi/api.github.com/operations/actions/list-workflow-runs.json#L69

### I expected

<details>
  <summary>"status" param can be one of the check run "status" or "conclusion"</summary>

```json
    {
      "name": "status",
      "description": "Returns workflow runs associated with the check run `status` or `conclusion` you specify. For example, a conclusion can be `success` or a status can be `completed`. For more information, see the `status` and `conclusion` options available in \"[Create a check run](https://developer.github.com/v3/checks/runs/#create-a-check-run).\"",
      "in": "query",
      "required": false,
      "schema": {
        "type": "string",
        "enum": [
          "queued",
          "in_progress",
          "completed",
          "success",
          "failure",
          "neutral",
          "cancelled",
          "timed_out",
          "action_required"
        ]
      }
    },
```
</details>

**Comment by gr2m** [comment]
Did you by chance test if the status supports enum values from both check run status & conclusion?

**Comment by miyajan** [comment]
Yes, I tested. When I specified "completed", list workflow runs returned both successful and failed runs. And when I specified "success", it returned only successful runs. It seems the status parameter supports enum values from both check run status & conclusion.

**Comment by gr2m** [comment]
Thanks Miyata, I've shortly talked to the docs team. They are on a week-long summit right now so it might take a moment to be addressed, but I'm sure it will be resolved soon. I'll keep you posted

**Comment by miyajan** [comment]
Thank you for your quick response😆

**Comment by miyajan** [comment]
@gr2m Any update on this issue?

**Comment by gr2m** [comment]
Sorry I dropped the ball on this one. Is this a blocker for you in some way?

**Comment by miyajan** [comment]
This is not blocker for me because I execute the API with a way other than octokit now. But I don't like this workaround, so please fix it.🙇‍♂️
